kazoo/tests/test_partitioner.py

```diff
@@ -8,85 +9,140 @@
 
 
 class KazooPartitionerTests(KazooTestCase):
+    @staticmethod
+    def make_event():
+        return threading.Event()
+
     def setUp(self):
         super(KazooPartitionerTests, self).setUp()
         self.path = "/" + uuid.uuid4().hex
+        self.__partitioners = []
 
     def test_party_of_one(self):
-        partitioner = self.client.SetPartitioner(
-            self.path, set=(1, 2, 3), time_boundary=0.2)
-        partitioner.wait_for_acquire(14)
-        eq_(partitioner.state, PartitionState.ACQUIRED)
-        eq_(list(partitioner), [1, 2, 3])
-        partitioner.finish()
+        self.__create_partitioner(size=3)
+        self.__wait_for_acquire()
+        self.__assert_state(PartitionState.ACQUIRED)
+        self.__assert_partitions([0, 1, 2])
+        self.__finish()
```

Comment:
What's the rationale for using private class methods vs. just normal private methods?